src/scenarios/staticconsoletemplate/src/Program.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using System;
 
-namespace _50consoletemplate
+namespace staticconsoletemplate
```

Comment:
is staticconsoletemplate same as netcoreapp2.0 src ?  i dont see any diff between them except their names

Reply:
maybe we could just combine them?

Reply:
sure looks like it. I'll leave it to you in the SDK test refactoring.